machine-learning/app/schemas.py

```diff
@@ -0,0 +1,64 @@
+from pydantic import BaseModel
+
+
+def to_lower_camel(string: str) -> str:
+    tokens = [
+        token.capitalize() if i > 0 else token
+        for i, token in enumerate(string.split("_"))
+    ]
+    return "".join(tokens)
+
+
+class VisionModelRequest(BaseModel):
+    image_path: str
+
+    class Config:
+        alias_generator = to_lower_camel
+        allow_population_by_field_name = True
+
+
+class TextModelRequest(BaseModel):
+    text: str
+
+
+class TextResponse(BaseModel):
+    __root__: str
+
+
+class MessageResponse(BaseModel):
+    message: str
+
+
+class TagResponse(BaseModel):
+    __root__: list[str]
```

Comment:
I assume `__root__` here mean the `TagReponse` model itself is considered as a list of `str`, correct?

Reply:
Yup, it's basically a type alias.